samples/CallWithChat/public/index.html

```diff
@@ -8,6 +8,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="description" content="Azure Communication Services - UI Library: Call With Chat Sample app" />
     <title>UI Library Call With Chat Sample</title>
+
+    <!-- For debugging purposes only. Remove for production builds. -->
```

Comment:
Is there a way to only inject this using webpack plugin - when it is production build, like --mode=production --env production, we remove this

Reply:
Technically would be possible, but I didn't walk down that road as I really want to have this in our deployed samples (and its probably relatively complicated). This comment is more for if Contoso copies this file they have a hit to remove this line